Restructure ieee directory

MarcHolness opened the pull request:
Restructure IEEE directory to align with Working Group direction.

Clean compile
$ ./check.sh

 Checking IEEE modules in directory /c/Users/mholness/Documents/GitHub/yang-1/standard/ieee
cd /c/Users/mholness/Documents/GitHub/yang-1/standard/ieee

 Checking modules with pyang in 802.1 :

pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1ab-lldp.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1ab-types.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1ax.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-ats.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-bridge.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-cfm.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-cfm-bridge.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-cfm-mip.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-cfm-types.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-pb.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-stream-filters-gates.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-tpmr.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-tsn-types.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-types.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1q-vlan-bridge.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-dot1x.yang

 Checking modules with pyang in 802.3 :

pyang --verbose ../ietf/RFC/ -p draft/ ieee802-ethernet-interface.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-ethernet-interface-half-duplex.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-ethernet-link-oam.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-ethernet-pon.yang
pyang --verbose ../ietf/RFC/ -p draft/ ieee802-ethernet-pse.yang

MarcHolness commented:
Hi Einar,


How do I go about bring the PR up to date with the master branch?


Marc.



________________________________
From: Einar Nilsen-Nygaard <notifications@github.com>
Sent: Saturday, December 1, 2018 3:57 PM
To: YangModels/yang
Cc: Holness, Marc; Author
Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory (#512)


@einarnn requested changes on this pull request.

Can you bring the PR up to date with the master branch?

Cheers,

Einar

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/YangModels/yang/pull/512#pullrequestreview-180538053>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOUty5IONDE_NxB3p6yfRlBgdhBLd6qsks5u0u0-gaJpZM4Y8NjH>.

einarnn commented:
Marc,

Take a look at this how-to from GitHub:

https://help.github.com/articles/syncing-a-fork/

In this case, "upstream" would be set up like this:

> git remote add upstream https://github.com/YangModels/yang.git

Cheers,

Einar
On 2 Dec 2018, 22:33 +0000, Marc Holness <notifications@github.com>, wrote:
>
> Hi Einar,
>
>
> How do I go about bring the PR up to date with the master branch?
>
>
> Marc.
>
>
>
> ________________________________
> From: Einar Nilsen-Nygaard <notifications@github.com>
> Sent: Saturday, December 1, 2018 3:57 PM
> To: YangModels/yang
> Cc: Holness, Marc; Author
> Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory (#512)
>
>
> @einarnn requested changes on this pull request.
>
> Can you bring the PR up to date with the master branch?
>
> Cheers,
>
> Einar
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<https://github.com/YangModels/yang/pull/512#pullrequestreview-180538053>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOUty5IONDE_NxB3p6yfRlBgdhBLd6qsks5u0u0-gaJpZM4Y8NjH>.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

MarcHolness commented:
Hi Einar,

I think I have fixed the merger issue (i.e., conflict). I had to update ieee802-dot1ab-lldp.yang. I was not using the most recent version before. I am now.

Regards,

Marc.

einarnn commented:
Marc,

Are you going to resubmit the structural changes you wanted to make? Or what?

Cheers,

Einar
On 3 Dec 2018, 15:17 +0000, Marc Holness <notifications@github.com>, wrote:
> Hi Einar,
> I think I have fixed the merger issue (i.e., conflict). I had to update ieee802-dot1ab-lldp.yang. I was not using the most recent version before. I am now.
> Regards,
> Marc.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.

einarnn commented:
Suggest you make sure you have updated your fork to be up-to-date with the
master. In fact, I would tend to create a new fork if possible.

Cheers,

Einar

On Mon, 10 Dec 2018 at 15:29, Marc Holness <notifications@github.com> wrote:

>
> Yes. Let me resubmit. I will do this in an hour or so.
>
> Marc.
>
> From: Einar Nilsen-Nygaard <notifications@github.com>
> Sent: Monday, December 10, 2018 10:06 AM
> To: YangModels/yang <yang@noreply.github.com>
> Cc: Holness, Marc <mholness@ciena.com>; Author <author@noreply.github.com>
>
> Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory
> (#512)
>
> Marc,
>
> Are you going to resubmit the structural changes you wanted to make? Or
> what?
>
> Cheers,
>
> Einar
> On 3 Dec 2018, 15:17 +0000, Marc Holness <notifications@github.com<mailto:
> notifications@github.com>>, wrote:
> > Hi Einar,
> > I think I have fixed the merger issue (i.e., conflict). I had to update
> ieee802-dot1ab-lldp.yang. I was not using the most recent version before. I
> am now.
> > Regards,
> > Marc.
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<
> https://github.com/YangModels/yang/pull/512#issuecomment-445847416>, or
> mute the thread<
> https://github.com/notifications/unsubscribe-auth/AOUtyxXV1Cv7rsM14jxXb_2Vfviy_Tynks5u3nhbgaJpZM4Y8NjH>.
>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/YangModels/yang/pull/512#issuecomment-445856205>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFKGcBbJ-N_lvpo4kNh1rkLpq9LvpIa2ks5u3n3TgaJpZM4Y8NjH>
> .
>


-- 
Einar Nilsen-Nygaard
It is a mistake to think you can solve any major problems just with
potatoes. --Douglas Adams

MarcHolness commented:
Okay. Good suggestion.

Marc.

From: Einar Nilsen-Nygaard <notifications@github.com>
Sent: Monday, December 10, 2018 10:42 AM
To: YangModels/yang <yang@noreply.github.com>
Cc: Holness, Marc <mholness@ciena.com>; Author <author@noreply.github.com>
Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory (#512)

Suggest you make sure you have updated your fork to be up-to-date with the
master. In fact, I would tend to create a new fork if possible.

Cheers,

Einar

On Mon, 10 Dec 2018 at 15:29, Marc Holness <notifications@github.com<mailto:notifications@github.com>> wrote:

>
> Yes. Let me resubmit. I will do this in an hour or so.
>
> Marc.
>
> From: Einar Nilsen-Nygaard <notifications@github.com<mailto:notifications@github.com>>
> Sent: Monday, December 10, 2018 10:06 AM
> To: YangModels/yang <yang@noreply.github.com<mailto:yang@noreply.github.com>>
> Cc: Holness, Marc <mholness@ciena.com<mailto:mholness@ciena.com>>; Author <author@noreply.github.com<mailto:author@noreply.github.com>>
>
> Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory
> (#512)
>
> Marc,
>
> Are you going to resubmit the structural changes you wanted to make? Or
> what?
>
> Cheers,
>
> Einar
> On 3 Dec 2018, 15:17 +0000, Marc Holness <notifications@github.com<mailto:
<mailto:notifications@github.com%3cmailto:%0b>> notifications@github.com<mailto:notifications@github.com>>>, wrote:
> > Hi Einar,
> > I think I have fixed the merger issue (i.e., conflict). I had to update
> ieee802-dot1ab-lldp.yang. I was not using the most recent version before. I
> am now.
> > Regards,
> > Marc.
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<
> https://github.com/YangModels/yang/pull/512#issuecomment-445847416>, or
> mute the thread<
> https://github.com/notifications/unsubscribe-auth/AOUtyxXV1Cv7rsM14jxXb_2Vfviy_Tynks5u3nhbgaJpZM4Y8NjH>.
>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/YangModels/yang/pull/512#issuecomment-445856205>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFKGcBbJ-N_lvpo4kNh1rkLpq9LvpIa2ks5u3n3TgaJpZM4Y8NjH>
> .
>


--
Einar Nilsen-Nygaard
It is a mistake to think you can solve any major problems just with
potatoes. --Douglas Adams


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/YangModels/yang/pull/512#issuecomment-445860818>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOUtyweBetP5EOQzgo5NcRVGLGccB6Jbks5u3oDRgaJpZM4Y8NjH>.

MarcHolness commented:
Hi Einar,

Okay, created new fork. Resubmitted IEEE directory restructuring changes. No YANG modules changed, just moved to other directories.

Regards,

Marc.

From: Einar Nilsen-Nygaard <notifications@github.com>
Sent: Monday, December 10, 2018 10:42 AM
To: YangModels/yang <yang@noreply.github.com>
Cc: Holness, Marc <mholness@ciena.com>; Author <author@noreply.github.com>
Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory (#512)

Suggest you make sure you have updated your fork to be up-to-date with the
master. In fact, I would tend to create a new fork if possible.

Cheers,

Einar

On Mon, 10 Dec 2018 at 15:29, Marc Holness <notifications@github.com<mailto:notifications@github.com>> wrote:

>
> Yes. Let me resubmit. I will do this in an hour or so.
>
> Marc.
>
> From: Einar Nilsen-Nygaard <notifications@github.com<mailto:notifications@github.com>>
> Sent: Monday, December 10, 2018 10:06 AM
> To: YangModels/yang <yang@noreply.github.com<mailto:yang@noreply.github.com>>
> Cc: Holness, Marc <mholness@ciena.com<mailto:mholness@ciena.com>>; Author <author@noreply.github.com<mailto:author@noreply.github.com>>
>
> Subject: [**EXTERNAL**] Re: [YangModels/yang] Restructure ieee directory
> (#512)
>
> Marc,
>
> Are you going to resubmit the structural changes you wanted to make? Or
> what?
>
> Cheers,
>
> Einar
> On 3 Dec 2018, 15:17 +0000, Marc Holness <notifications@github.com<mailto:
<mailto:notifications@github.com%3cmailto:%0b>> notifications@github.com<mailto:notifications@github.com>>>, wrote:
> > Hi Einar,
> > I think I have fixed the merger issue (i.e., conflict). I had to update
> ieee802-dot1ab-lldp.yang. I was not using the most recent version before. I
> am now.
> > Regards,
> > Marc.
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub, or mute the thread.
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub<
> https://github.com/YangModels/yang/pull/512#issuecomment-445847416>, or
> mute the thread<
> https://github.com/notifications/unsubscribe-auth/AOUtyxXV1Cv7rsM14jxXb_2Vfviy_Tynks5u3nhbgaJpZM4Y8NjH>.
>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/YangModels/yang/pull/512#issuecomment-445856205>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFKGcBbJ-N_lvpo4kNh1rkLpq9LvpIa2ks5u3n3TgaJpZM4Y8NjH>
> .
>


--
Einar Nilsen-Nygaard
It is a mistake to think you can solve any major problems just with
potatoes. --Douglas Adams


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/YangModels/yang/pull/512#issuecomment-445860818>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AOUtyweBetP5EOQzgo5NcRVGLGccB6Jbks5u3oDRgaJpZM4Y8NjH>.
